MAVProxy/modules/mavproxy_magical/data/arrow.mtl

```diff
@@ -0,0 +1,14 @@
+# Magical Arrow Material (c) by Intel Corporation
+#
+# Magical Arrow Material is licensed under a
+# Creative Commons Attribution-ShareAlike 4.0 International License.
+#
+# You should have received a copy of the license along with this
```

Comment:
Yep, makes sense. I used templates from https://creativecommons.org/2014/01/07/plaintext-versions-of-creative-commons-4-0-licenses/ .

Maybe ship them with the project? Is there a way to have setup.py doing that for us? I'm not used to writing python distributions.